lib/livebook/hubs/dockerfile.ex

```diff
@@ -289,6 +290,48 @@ defmodule Livebook.Hubs.Dockerfile do
     config.zta_provider != nil and config.zta_key != nil
   end
 
+  @doc """
+  Returns information for deploying Livebook Agent using Docker.
+  """
+  @spec agent_docker_info(config(), Hubs.Provider.t(), Livebook.Teams.AgentKey.t()) :: %{
+          image: String.t(),
+          env: list({String.t(), String.t()})
+        }
+  def agent_docker_info(config, %Hubs.Team{} = hub, agent_key) do
```

Comment:
Nitpick on the naming:

* build_dockerfile -> airgapped_dockerfile (or offline_dockerfile but airgapped is better as it is not deployment group specific)
* agent_docker_info -> online_docker_info
* warnings -> airgapped_warnings (or offline)
* agent_warnings -> online_warnings